hooks/commit-msg.sample

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
It might as well have been written in Python, given that `bash` is not guaranteed to be installed on all systems. Either rewrite it in POSIX shell, or in a proper scripting language.